types/web-element.d.ts

```diff
@@ -149,6 +149,8 @@ export interface ScopedElement extends Element, PromiseLike<WebElement> {
 
   getText(): ElementValue<string>;
 
+  text(): ElementValue<string>;
+
```

Comment:
Can you also club the related commands together over here (in all PRs)?

For ex., over here there shouldn't be an extra line between `getText()` and `text()` commands.